crates/block-producer/src/batch_builder/batch.rs

```diff
@@ -134,14 +182,14 @@ impl TransactionBatch {
         })
     }
 
-    /// Returns unauthenticated input notes set consumed by the transactions in this batch.
-    pub fn unauthenticated_input_notes(&self) -> &BTreeSet<NoteId> {
-        &self.unauthenticated_input_notes
+    /// Returns input notes list consumed by the transactions in this batch.
+    pub fn input_notes(&self) -> &[InputNoteCommitment] {
```

Comment:
I would also mention in the comment that any unauthenticated input notes which have matching output notes within this batch are not included in this list.